src/components/molecules/Modal/Modal.test.tsx

```diff
@@ -3,10 +3,22 @@ import { render, waitForElement } from 'react-testing-library';
 import Modal from './Modal';
 
 describe('<Modal />', () => {
-  it('render the opened Modal', async () => {
+  it('can render un-opened', async () => {
+    Modal.setAppElement('body');
+    const { queryByText } = render(<Modal isOpen={false}>text</Modal>);
+
+    expect(queryByText('text')).toBe(null);
+  });
+
+  it('can render opened', async () => {
     Modal.setAppElement('body');
     const { getByText } = render(<Modal isOpen>text</Modal>);
+
     const modal = await waitForElement(() => getByText('text'));
     expect(modal).toMatchSnapshot();
   });
+
+  it('allows to set global styles', () => {
```

Comment:
I had to a some more tests as **codecov** wouldn't allow to merge this due to loosing coverage...

It seems [testing the ouptut of `createGlobalStyles`](https://spectrum.chat/styled-components/jest/testing-a-createglobalstyle~7ed712f0-e0f1-430b-85c9-bcc9aa23e077) is no really straightforward either 😐